railties/lib/rails/generators/rails/app/templates/package.json.tt

```diff
@@ -2,10 +2,10 @@
   "name": "<%= app_name.underscore.dasherize %>",
   "private": true,
   "dependencies": {
-    "@rails/ujs": "^6.0.0"<% unless options[:skip_turbolinks] %>,
+    "@rails/ujs": "^<%= npm_version %>"<% unless options[:skip_turbolinks] %>,
```

Comment:
What do you think about matching the exact version at the same time (as noted in https://github.com/rails/rails/issues/42012)?

```suggestion
    "@rails/ujs": "<%= npm_version %>"<% unless options[:skip_turbolinks] %>,
```

Reply:
I think we should update the App Updater too. I can help with that if you like.

Reply:
@ghiculescu Good idea! Feel free to commit to this branch or if you want to open another PR is fine w/ me too :bow:

I think removing the `^` is a good idea, also we can add a condition for `--main` to just use `latest` tag for that edge case.

Reply:
I don’t have commit access, but if you give me access to your fork, can do.

Reply:
@ghiculescu done! 🙏